certora/specs/ghoVariableDebtToken.spec

```diff
@@ -485,17 +498,17 @@ rule integrityOfBurn_userIsolation() {
 * Integrity of updateDiscountDistribution
 ***************************************************************/
 
-/**
-* @title proves that the discount rate is calculated correctly when calling updateDiscountDistribution
-**/
+// /**
+// * @title proves that the discount rate is calculated correctly when calling updateDiscountDistribution
+// **/
```

Comment:
Does the rule finish successfully? If it times out it should be excluded from CI?

Reply:
it's commented out and not included in CI